.github/CODEOWNERS

```diff
@@ -113,17 +110,17 @@
 
 
 # Core documentation
-/AUTHORS @seanlip
-/CHANGELOG @seanlip
-/CONTRIBUTORS @seanlip
-/LICENSE @seanlip
-/NOTICE @seanlip
+/AUTHORS @BenHenning
```

Comment:
Do we need a TODO comment here? (Or is it a permanent change?)

Reply:
Thought about it more, and this led to a conversation with @BenHenning. Made further updates accordingly. Thanks again for flagging!